src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -107,13 +110,22 @@ public HostStatus GetHostStatus()
                     status.Errors.Add(Utility.FlattenException(lastError));
                 }
 
-                return status;
+                var parameters = Request.GetQueryParameterDictionary();
+                string value = null;
+                if (parameters.TryGetValue(ScriptConstants.CheckLoadQueryParameterName, out value) && value == "1")
+                {
+                    status.Load = new LoadStatus
```

Comment:
While we only have a simple bool to return now, I'm defining the payload to be extensible for the future. E.g. we'll likely want to return additional details in the future. To do so, we'll add members to LoadStatus.